.gitignore

```diff
@@ -7,3 +7,4 @@ dist/
 .coverage
 .coverage.xml
 .venv
+venv
```

Comment:
Will just `*venv/` work?

Reply:
So it's inline with githubs recommended ignore: 
https://github.com/github/gitignore/blob/4488915eec0b3a45b5c63ead28f286819c0917de/Python.gitignore#L122C2-L122C2 I altered it slightly.

Your suggestion does work though 😄

Reply:
In that case, your current change should be OK.